examples/combined_ODNP.py

```diff
@@ -167,61 +164,53 @@
 ini_time = time.time()
 vd_data = None
 for vd_idx, vd in enumerate(vd_list_us):
+    # call A to run_IR
```

Comment:
I saw this up above too... why?

Reply:
I label the calls to IR so that I can refer to them in the comments

Reply:
and so when I copy/paste them to compare, I'm sure of what I'm looking at